lib/signs/utilities/messages.ex

```diff
@@ -33,12 +33,24 @@ defmodule Signs.Utilities.Messages do
           {Content.Message.Empty.new(), Content.Message.Empty.new()}
 
         sign_config == :headway ->
-          get_headway_or_alert_messages(sign, current_time, alert_status)
+          get_alert_messages(alert_status, sign) ||
+            Signs.Utilities.Headways.get_messages(sign, current_time)
+
+        sign_config == :temporary_terminal and
+            alert_status in [:shuttles_transfer_station, :suspension_transfer_station] ->
+          case Signs.Utilities.Predictions.get_messages(predictions, sign) do
+            {%Content.Message.Empty{}, %Content.Message.Empty{}} ->
+              Signs.Utilities.Headways.get_messages(sign, current_time)
```

Comment:
Ah yeah good point about the RJFK case, we'd lose the the platform prediction formatting so I'll move the logic to the `true` path. I don't think it'll be as relevant to mezzanine headways more broadly though (I assume you're referring to paging headways?). Since the underlying assumption is that the stations are temporary terminals, we know we'll only be concerned with predictions/headways in a single direction so there's no use case for paging headways.